.github/workflows/build.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [1.18, 1.19]
```

Comment:
Since it’s yaml, `1.20` is recognized as a number `1.2` instead of a string. Therefore, we should specify it as a string `'1.20'`.

Reply:
Will resolve it by including versions in quotes